src/test/java/pl/bajerska/befindyourmeal/user/UserServiceTests.java

```diff
@@ -0,0 +1,118 @@
+package pl.bajerska.befindyourmeal.user;
+
+import org.junit.Rule;
+import org.junit.jupiter.api.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+import pl.bajerska.befindyourmeal.exception.ExceptionAdviceSet;
+import pl.bajerska.befindyourmeal.exception.InvalidUserPasswordException;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class UserServiceTests {
+    @Test
+    public void shouldNotFindUser() {
+
+
+        UserRepository repository = Mockito.mock(UserRepository.class);
+        UserServiceImpl service = new UserServiceImpl(repository);
+        Mockito.when(repository.findByUsername("zofia.bajerska")).thenReturn(null);
+        String name = new String("zofia.bajerska@gmail.com");
```

Comment:
These are different strings, so you get null, but not because it was tested. ;)
You can always verify (Mockito.verify) if the expected method was called with the expected arguments (but please, it's another PR!)